doc/ale-ruby.txt

```diff
@@ -14,6 +14,25 @@ g:ale_ruby_brakeman_options                       *g:ale_ruby_brakeman_options*
 
 
 ===============================================================================
+rails_best_practices                            *ale-ruby-rails_best_practices*
+
+g:ale_ruby_rails_best_practices_executable *g:ale_ruby_rails_best_practices_executable*
```

Comment:
Write your lines likes so instead:

```vim
g:foo
         *g:foo*
```

Only with the last character aligned to the right margin. `:ALEFixSuggest` will suggest a function for aligning your help tags to the right margin for you, which you can run with `:ALEFix`.